j5/boards/sr/v4/motor_board.py

```diff
@@ -0,0 +1,55 @@
+"""Classes for the SR v4 Motor Board."""
+from typing import TYPE_CHECKING, List, Optional, Type, cast
+
+from j5.backends import Backend
+from j5.boards import Board
+from j5.components.motor import Motor, MotorInterface, MotorSpecialState
+
+if TYPE_CHECKING:  # pragma: no cover
+    from j5.components import (  # noqa: F401
+        Component,
+    )
+
+
+class MotorBoard(Board):
+    """Student Robotics v4 Motor Board."""
+
+    def __init__(self, serial: str, backend: Backend):
+        self._serial = serial
+        self._backend = backend
+
+        self._outputs: List[Motor] = [
+            Motor(output, self, cast(MotorInterface, self._backend))
+            for output in range(0, 2)
+        ]
+
+    @property
+    def name(self) -> str:
+        """Get a human friendly name for this board."""
+        return "Student Robotics v4 Motor Board"
+
+    @property
+    def serial(self) -> str:
+        """Get the serial number."""
+        return self._serial
+
+    @property
+    def firmware_version(self) -> Optional[str]:
+        """Get the firmware version of the board."""
+        return self._backend.get_firmware_version()
```

Comment:
I'd recommend trying to get this is a property, like the thing it's in ;)

Reply:
Opened #236. Leaving this as is to match existing code style.